backoffice/src/Views/Shared/Droit/_DroitListTable.cshtml

```diff
@@ -55,6 +58,9 @@
                             }
                         }
                     </td>
+                    <td class="p-3 text-uppercase" style="text-align: left;">
```

Comment:
Likewise with this inline styling, ideally anything we want should be achieved by bootstrap classes